data.tf

```diff
@@ -0,0 +1,38 @@
+data "aws_iam_policy_document" "role_trust_relationship" {
+  statement {
+    sid = "1"
+
+    effect = "Allow"
+    actions = [
+      "sts:AssumeRole",
+      "sts:TagSession",
+      "sts:SetSourceIdentity"
+    ]
+
+    principals {
+      identifiers = ["rolesanywhere.amazonaws.com"]
+      type        = "Service"
+    }
+
+    condition {
+      test     = "StringEquals"
+      values   = var.x509_subject_ou_values
+      variable = "aws:PrincipalTag/x509Subject/OU"
+    }
+
+    condition {
+      test     = "StringEquals"
+      values   = var.x509_subject_cn_values
+      variable = "aws:PrincipalTag/x509Subject/CN"
+    }
+  }
+
+  dynamic "statement" {

```

Comment:
I wonder if we shouldn't make this statement a separate policy that we attach to the role? Because if it's here I think it wouldn't be in the permissions, it would be put in trust relationships

Reply:
We spoken out this and it should be fine.